File: sistemas/mobile/ohara/src/app/model/review.ts
import { Usuario } from "./usuario";

export class Review {
    idReview: number | null;
    usuario: Usuario | null;
    idUsuario: number;
    idManga: number | null;
    descricao: string;

    constructor() {
        this.idReview = null;
        this.usuario = new Usuario();
        this.idUsuario = 0;
        this.idManga = 0;
        this.descricao = "";
    }
}

File: api/ohara/src/main/java/br/com/ohara/model/Review.java
package br.com.ohara.model;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import lombok.Data;

@Data
public class Review {
	private Long idReview;
	
	@NotNull
	private Usuario usuario;
	
	@NotNull
	private Long idManga;
	
	@NotBlank
	@Size(max = 500, message = "A descricao não pode exceder 500 caracteres")
	private String descricao;

}
